src/test/java/org/javawebstack/orm/test/shared/knowledge/QueryKnowledgeBase.java

```diff
@@ -0,0 +1,36 @@
+package org.javawebstack.orm.test.shared.knowledge;
+
+import java.util.Arrays;
+import java.util.HashSet;
+
+public class QueryKnowledgeBase {
+
+    public static HashSet<String> sectionNames;
```

Comment:
Constants should be final and the naming convention is all uppercase with underscores (see https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html)

Reply:
Thank you. Change accordingly to `QUOTE_CHARACTERS`.